docs/tests.rst

```diff
@@ -110,6 +110,57 @@ Via Docker with a graphical instance of Firefox (macOS)
 
   $ docker-compose -f docker-compose.test.yml run --rm acceptance
 
+How to Debug the Selenium Tests on Docker
+-----------------------------------------
+In order to debug code in the `web` container with the Docker-Selenium test suite it is necessary to use `remote-pdb`_.
+
+1. First install RemotePdb in the web container. Run the web container with the bash command attached:
+
+.. code-block:: bash
+
+  $ docker-compose run web bash
+
+Secondly install with the pip command the `remotePdb` package:
+
+.. code-block:: bash
+
+  (virtualenv) $ pip install remote-pdb
+
+Finally exit from the virtualenv and from the docker container with `ctrl + d`.
+
+2. Find a spot where you would like tracing to begin, and insert the following code:
+
+.. code-block:: python
+
+  from remote_pdb import RemotePdb
+  RemotePdb('0.0.0.0', 4444).set_trace()
+
+3. Run the acceptance tests with `docker-compose`
+
+.. code-block:: bash
+
+  $ docker-compose -f docker-compose.test.yml run --rm acceptance
+
+4. Write down the IP address of the web container because you will need it later:
+
+.. code-block:: bash
+
+  $ docker inspect inspirenext_web_1
+  {
+    [...]
+    "IPAddress": "123.456.7.890",
+    [...]
+  }
```

Comment:
The output of `inspect` is quite massive. Instead of hunting for it with your eyes you can `grep` for it: `docker inspect inspirenext_web_1 | grep IPAddress`